static/src/components/EndScreen.js

```diff
@@ -0,0 +1,176 @@
+import React, { PropTypes, Component } from 'react';
+import { RaisedButton, FlatButton, Dialog, Table, TableBody,
+         TableRow, TableRowColumn, TableHeader, tableHeaderColumn } from 'material-ui';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as actions from '../actions/game';
+
+function mapStateToProps(state) {
+    return {
+        game: state.game.game,
+        started: state.game.started,
+        error: state.game.error,
+        loading: state.game.loading,
+        playerCount: state.game.playerCount,
+    };
+}
+
+function mapDispatchToProps(dispatch) {
+    return bindActionCreators(actions, dispatch);
+}
+
+export class EndScreen extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            userID: null,
+            displayData: null,
+            fetch: true,
+            open: false,
+        };
+        if (this.props.game) {
+            this.state.userData = this.props.game.player;
+            this.state.allPlayers = this.props.game.allPlayers;
+            this.getData();
+        }
+    }
+
+    componentDidMount() {
+        console.log('componentDidMount');
+    }
+
+    getData() {
```

Comment:
You don't need to request this as the game object is mapped to the components props via redux